src/assets/scripts/airport/Arrival/ArrivalBase.js

```diff
@@ -301,20 +513,21 @@ export default class ArrivalBase {
 
     /**
      * Determine delay until next spawn
+     *
+     * @for ArrivalBase
+     * @method nextInterval
+     * @return {number}
      */
     nextInterval() {
-        // TODO: these next 3 vars could easily be moved to a constants file
-        // nautical miles
-        const min_entrail = 5.5;
-        // in seconds
-        const min_interval = min_entrail * (3600 / this.speed);
-        let tgt_interval = 3600 / this.frequency;
```

Comment:
so this is actually `ONE_HOUR_IN_SECONDS = 3600`?

That makes sense.  Time is hard, especially in javascript where time collides with the browser and a current user's timezone settings.  Often times you will see time handled in milliseconds, which is what I had assumed.  I'm glad I asked because I guessed wrong.